CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Change Log
 ## [Releases](https://github.com/NetApp/harvest/releases)
 
+
+## 23.05.0 / 2023-05-??
+
+**IMPORTANT** The `volume_aggr_labels` metric is being deprecated in this
```

Comment:
**IMPORTANT** The `volume_aggr_labels` metric is being deprecated in the `23.05`
release and will be removed in the `23.08` release of Harvest ([#1966](https://github.com/NetApp/harvest/pull/1966))